### PR TITLE
Compute beta normalized for both old and new C-MOD shots

### PR DESCRIPTION
**Pull request by yumouwei**
# Original problems

- `\efit_aeqdsk:betan` is undefined for pre-2000 shots.
- There's a separate `betan` computation in `cmod/efit.py` for pre-2000 shots written by someone, but the calculation is wrong.

# Implemented changes

- Adding `get_beta_normalized` function in `cmod/physics.py`. This method takes in beta_t, ip, aminor, & btor data from EFIT and computes beta_n using the same formulation as `\efit_aeqdsk:betan`. The output agrees with `\efit_aeqdsk:betan` for post-2000 shots.
- Remove existing `betan` fetching & computation in `cmod/efit.py`.
- Update CMOD output shape in `examples/efit.py`.

### CODE REVIEW

File: disruption_py/machine/cmod/efit.py [hunk 1]
```diff
@@ -130,14 +128,6 @@ def get_efit_parameters(params: PhysicsMethodParams):
                 print("unable to get v_loop_efit")
                 efit_data["v_loop_efit"] = np.full(len(efit_time), np.nan)
 
-            # Compute beta_n
```
Comment (gtrevisan) [inline disruption_py/machine/cmod/efit.py:133]: there was another comment about `beta_n` just before the `if` statement acting on pre-2000 shots.

File: disruption_py/machine/cmod/physics.py [hunk 2]
```diff
@@ -1754,6 +1754,64 @@ def get_sxr_data(params: PhysicsMethodParams):
         sxr = interp1(t_sxr, sxr, params.times)
         return {"sxr": sxr}
 
+    @staticmethod
+    @physics_method(
```
Comment (gtrevisan) [inline disruption_py/machine/cmod/physics.py:1758]: you should be able to have these decorator arguments in a single line without black complaining.

File: disruption_py/machine/cmod/physics.py [hunk 3]
```diff
@@ -1754,6 +1754,64 @@ def get_sxr_data(params: PhysicsMethodParams):
...
+        for the plasma current and toroidal field data.
+
+        Parameters
+        ----------
+        params : PhysicsMethodParams
+            The parameters containing the MDSplus connection, shot id and more.
+
+        Returns
+        -------
+        beta_n: the normalized beta given in percentage.
```
Comment (gtrevisan) [inline disruption_py/machine/cmod/physics.py:1783]: I'm not sure how we prepared the other docstrings, but we return a dict which holds betan, not betan itself.

**Review by gtrevisan — CHANGES_REQUESTED** [review]
LGTM! minor comments.

**Comment by gtrevisan** [comment]
@crea-psfc is it worth analyzing the same for DIII-D, as well?

**Comment by crea-psfc** [comment]
not aware of similar issues, but it's worth it to do a quick check. IMHO, EFIT is better maintained at DIII-D

File: disruption_py/machine/cmod/physics.py (hunk 2 above)
Comment (yumouwei) [reply disruption_py/machine/cmod/physics.py:1758]: I tried but it doesn't seem to work.

Comment (gtrevisan) [reply disruption_py/machine/cmod/physics.py]: I tried it and it did work. :smile:

**Review by gtrevisan — APPROVED** [review]

**Pull request merged**